src/Entity/Product.php

```diff
@@ -0,0 +1,26 @@
+<?php
```

Comment:
Code style is messed up a bit, use autoformat functionality of your IDE (phpStorm maybe?).

Reply:
This file is not used. It's gonna be deleted in the next commit.